data.tf

```diff
@@ -0,0 +1,62 @@
+locals {
+  fqdn      = "${element(concat(data.aws_efs_file_system.default.*.dns_name, list("")), 0)}"
```

Comment:
change `fqdn` to `efs_fqdn` below your output is `fqdn_efs`. Be consistent with what you choose :D